docker/scan-cli-plugin
# Mention more details about linux packages in README

**- What I did**

Mention more details about linux packages in README.md

**- How I did it**

Just edit the README.md file to add more details

**- How to verify it**

https://github.com/docker/scan-cli-plugin/issues/164#issuecomment-861319137

Check the packages here: https://download.docker.com/linux/ubuntu/dists/bionic/pool/stable



cc docker/scan-cli-plugin#140 @thaJeztah
